src/main/java/sirius/biz/process/ProcessEnvironment.java

```diff
@@ -95,6 +95,11 @@ public void addTiming(String counter, long millis) {
 
     @Override
     public void addTiming(String counter, long millis, boolean adminOnly) {
+        // Drops the dollar sign used in NLS.smartGet, because the counters are always translated.
+        if (counter.startsWith("$")) {
+            counter = counter.substring(1);
```

Comment:
maybe return first if Strings.isEmpty(counter) to prevent NPE? , but idk throwing an error if counter is null might be the desireable outcome

Reply:
NPE here is pretty much a programming error, so an error is OK.
We could improve the error (instead of NPE) by annotating all addTiming/incCounter with @Nonnull. Not sure I wanna do that now 🤪